Packages/MIES/MIES_SweepFormula_Helpers.ipf

```diff
@@ -432,23 +432,25 @@ Function/WAVE SFH_GetSweepsForFormula(string graph, WAVE range, WAVE/Z selectDat
 				continue
 			endif
 			numEpochs = DimSize(epIndices, ROWS)
-			WAVE range = SFH_GetEmptyRange()
-			Redimension/N=(-1, numEpochs) range
+			WAVE adaptedRange = SFH_GetEmptyRange()
+			Redimension/N=(-1, numEpochs) adaptedRange
 			for(j = 0; j < numEpochs; j += 1)
 				epIndex = epIndices[j]
-				range[0][j] = str2num(epochInfo[epIndex][EPOCH_COL_STARTTIME]) * ONE_TO_MILLI
-				range[1][j] = str2num(epochInfo[epIndex][EPOCH_COL_ENDTIME]) * ONE_TO_MILLI
+				adaptedRange[0][j] = str2num(epochInfo[epIndex][EPOCH_COL_STARTTIME]) * ONE_TO_MILLI
+				adaptedRange[1][j] = str2num(epochInfo[epIndex][EPOCH_COL_ENDTIME]) * ONE_TO_MILLI
 			endfor
 		else
-			Redimension/N=(-1, 1) range
+			Duplicate/FREE range, adaptedRange
```

Comment:
Please add a note to the function description that the passed range should be treated as const.